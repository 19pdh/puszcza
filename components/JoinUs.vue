<template>
  <section class="joinus">
    <div class="heading">
      <div class="title">
        <div class="text">Rozpocznij swoją harcerską przygodę!</div>
      </div>
      <plain-button @click="$router.push('/kontakt')" text="Dołącz do nas" />
    </div>
  </section>
</template>

<script>
import PlainButton from './Buttons/PlainButton'

export default {
  name: 'JoinUs',
  components: {
    PlainButton
  }
}
</script>

<style scoped>
.joinus {
  width: 100%;

  padding: 80px 20%;

  background-image: url('/assets/bg.jpg');
  background-size: cover;
  background-position: center;
  background-repeat: no-repeat;

  display: flex;
  flex-direction: row;
  justify-content: center;
}

.heading {
  width: 900px;
  padding: 50px 0 150px 0;

  display: flex;
  flex-direction: column;
}

.title {
  background-color: #340c0e;
  color: #ffffff;

  width: 260px;
  height: 85px;
  margin-bottom: 20px;

  text-align: left;
}

.title .text {
  margin: 0;
  padding: 10px;
  font-size: 24px;
}

.button {
  background-color: #507b34;
  padding: 10px;
  max-width: 150px;
  color: #ffffff;
  text-align: center;
}

.button a {
  text-decoration: none;
}

@media (max-width: 500px) {
  .heading {
    margin: 0;
    align-items: center;
  }

  .joinus {
    padding: 80px 20px;
  }
}
</style>