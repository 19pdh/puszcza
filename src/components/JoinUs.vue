<template>
  <section class="joinus">
    <div class="heading">
      <div class="title">
        <div class="text">Rozpocznij swoją harcerską przygodę!</div>
      </div>
      <div class="button">
        <router-link to="/kontakt">Dołącz do nas!</router-link>
      </div>
    </div>
  </section>
</template>

<style scoped>
.joinus {
  width: 100%;

  padding: 80px 20%;

  background: #dddddd;
}

.heading {
  max-width: 750px;
  padding: 100px 0;

  display: flex;
  flex-direction: column;
}

.title {
  background-color: #340c0e;
  color: #ffffff;

  width: 260px;
  height: 85px;
  margin-bottom: 20px;

  text-align: left;
}

.title .text {
  margin: 0;
  padding: 10px;
  font-size: 24px;
}

.button {
  background-color: #507b34;
  padding: 10px;
  max-width: 150px;
}

.button a {
  color: #ffffff;
  text-decoration: none;
}

@media (max-width: 500px) {
  .heading {
    margin: 0;
    align-items: center;
  }

  .joinus {
    padding: 80px 20px;
  }
}
</style>