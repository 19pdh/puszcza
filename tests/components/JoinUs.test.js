import { shallowMount } from '@vue/test-utils'
import JoinUs from '../../components/JoinUs'

describe('JoinUs', () => {
  // Now mount the component and you have the wrapper
  const wrapper = shallowMount(JoinUs)

  it('match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
