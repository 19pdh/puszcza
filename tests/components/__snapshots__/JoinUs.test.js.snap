// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JoinUs match snapshot 1`] = `
<section
  class="joinus"
>
  <div
    class="heading"
  >
    <div
      class="title"
    >
      <div
        class="text"
      >
        Rozpocznij swoją harcerską przygodę!
      </div>
    </div>
     
    <div
      class="button"
    >
      <router-link
        to="/kontakt"
      >
        Dołącz do nas!
      </router-link>
    </div>
  </div>
</section>
`;
